src/sentry/static/sentry/app/routes.jsx

```diff
@@ -215,7 +215,15 @@ function routes() {
         }
         component={errorHandler(LazyLoad)}
       />
-      <Route name="Alerts" path="alerts/">
+
+      <Route
+        name="Alerts"
+        path="alerts/"
+        component={errorHandler(LazyLoad)}
+        componentPromise={() =>
+          import(/* webpackChunkName: "ProjectAlerts" */ 'app/views/settings/projectAlerts')
```

Comment:
This is so we can more easily reuse the existing alert rules while we work on the header and other nav elements